src/css/main.less

```diff
@@ -905,3 +909,43 @@ fieldset[disabled] .btn-info.active {
     padding: .8em 1em; /* this would be "10px 0" otherwise */
   }
 }
+
+.tab-pane.active.flex-container {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  min-height: 100vh;
+
+  .wrapper {
+    display: flex;
+    flex: 1;
+
+    main {
+      flex: 1;
+      padding-left: (@base-spacing-unit * 2);
+    }
+
+    nav {
+      background: @sidebar-bg;
+      min-height: 100vh;
+      margin: -(@base-spacing-unit * 2.5) 0 0 -(@base-spacing-unit * 2);
+      order: -1;
```

Comment:
What's the advantage of placing the nav second in the markup and using `order` to rearrange them?